src/main/java/com/launchacademy/reviews/controllers/BerriesApiV1Controller.java

```diff
@@ -29,4 +32,14 @@ public ResponseEntity<Map<String, List<Berry>>> allBerries() {
     dataMap.put("berries", berryService.findAll());
     return new ResponseEntity<>(dataMap, HttpStatus.OK);
   }
-}
+
+  @GetMapping("/{id}")
+  public ResponseEntity<Map<String, Berry>> getBerry(@PathVariable Long id) {
+    Optional<Berry> berry = berryService.findById(id);
+    Map<String, Berry> dataMap = new HashMap<>();
+    if(berry.isPresent()) {
+      dataMap.put("berry", berry.get());
+    }
+    return new ResponseEntity<>(dataMap, HttpStatus.OK);
```

Comment:
In the cases where the berry is not present, let's send back a 404 with a message saying that this berry could not be found